modules/crm_core_activity/src/Entity/ActivityType.php

```diff
@@ -33,6 +33,12 @@
  *     "id" = "type",
  *     "label" = "name",
  *   },
+ *   config_export = {
+ *     "name",
+ *     "type",
+ *     "description",
+ *     "primary_fields",
```

Comment:
ActivityType doesn't seem to have a primary_fields property. I see it in the current schema.yml, but nowhere else.